.gitignore

```
@@ -6,3 +6,5 @@ build
 .coverage*
 .idea/*
 /dist
+**/*.sqlite
```

Does simply this work:

```suggestion
*.sqlite
```

---

Yup, sure!